packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -33,6 +34,8 @@ const {
 // cache is more likely to go stale than the images (which never go stale)
 // Note that the same image might be requested multiple times in the same run
 
+const validImageFormats = new Set([`jpg`, `png`, `webp`])
```

Comment:
Not sure why a set is needed here.
```suggestion
const validImageFormats = [`jpg`, `png`, `webp`]
```

Reply:
I was under the impression that set.has was significantly faster than array.includes

Reply:
It probably is but it doesn't matter on a 3 items list. For me when I read Sets I expect things to get added to it and making sure things are unique and that makes it more complex in my head.

Sets are fine, that's why I approved it but wanted to get the reason on the why :)

Reply:
It's just a pattern I always use for this sort of check, particularly when dealing with potentially large numbers of nodes, with a constant Set.